### PR TITLE
Removed numba requirement und bumped jax to 0.4.14 for native windows support

### PR DESCRIPTION
**Pull request by ThoreWietzke**
Removed numba as this was a constant source of installation errors. This was justified, as it was only used for one function which was never used in bayesnewton.

### CODE REVIEW

File: bayesnewton/utils.py [hunk 1]
```diff
@@ -4,11 +4,7 @@
 from jax.scipy.linalg import cholesky, cho_factor, cho_solve
 from jax.scipy.special import gammaln
 from jax.lax import scan
-# from matplotlib._png import read_png
```
Comment (wil-j-wil) [inline bayesnewton/utils.py:7]: can we keep this commented out import please

**Comment by wil-j-wil** [comment]
Thanks for this. One minor comment then we can merge

File: bayesnewton/utils.py (hunk 1 above)
Comment (ThoreWietzke) [reply bayesnewton/utils.py:7]: sure, already commited a fix

**Review by wil-j-wil — APPROVED** [review]
Thanks a lot

**Pull request merged**